src/components/calculator.js

```diff
@@ -1,71 +1,74 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './calculator.css';
 import './home.css';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { total: null, next: null, operation: null };
-  }
+const Calculator = () => {
+  // constructor(props) {
+  //   super(props);
+  //   this.state = { total: null, next: null, operation: null };
+  // }
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.%20They%20will%20make%20your%20project%20look%20messy.%20If%20you%20need%20to%20review%20a%20previous%20version%20of%20your%20code%2C%20you%20can%20always%20use%20git%20history.)